### ISSUE TITLE
Suggestion: Add a pump

### ISSUE THREAD

**Issue by BinaryVigilante** [CoFH/Feedback#943]
Hi!
Would it be possible to add a pump to Thermal Expansion? I would really love this feature!

Something like the pump from Buildcraft or Mekanism. And if it would use power when running, that would be an amazing plus!

Btw, I just want to say that I REALLY love this mod! I think it is great and I hope to see more amazing stuff added in the future!

Have a nice day/night!
Kind regards

**Comment by Griffdith** [comment]
Use the Aqueous Accumulator

**Comment by paul-soporan** [comment]
The acumulator doesn't work like a pump. It just creates water from adjacent water blocks. Tell me how would you collect huge quantities of lava from the nether with the aqueous aqumulator?

**Comment by Griffdith** [comment]
Didn’t;t know thats what he meant

**Comment by Ricktrandom** [comment]
It was kind of unclear, anyways if they haven't added a pump yet it would seem they're just avoiding it. I just make a resonant reservoir enchanted with holding then drink a fire res potion and wade through lava ocean until it's full. A Lava pump would kind of throw off the generators balance, since with this much lava in here it'd make too much sense to just use Magmatic Dynamos over all others if the process could be 100% automated.

**Comment by BinaryVigilante** [comment]
And what if you are using a modpack for example and the only industry mods are Thermal Expansion? But there are other fluids in the modpack that gets generated and you would like to pump them up? I think a pump in Thermal Expansion would be a great add-on.

**Comment by Griffdith** [comment]
Maybe Theraml Innovation?

**Comment by BinaryVigilante** [comment]
Hmm, looks interesting! I will try it and see how it works. Thanks!

**Comment by BinaryVigilante** [comment]
No, not really what I was looking for sadly... But thanks anyway! Still looks interesting ;)

**Comment by KingLemming** [comment]
Won't be in Thermal Expansion, but I do have plans to add something like a pump to a mod, yeah.

**Comment by BinaryVigilante** [comment]
Sounds very interesting! Can't wait ;)

**Comment by BinaryVigilante** [comment]
Sounds very interesting! Can't wait ;)
Thanks for your reply!

Have a great day!
Kind regards
